extensions/jackson/deployment/src/main/java/org/apache/camel/quarkus/component/jackson/JacksonProcessor.java

```diff
@@ -39,7 +39,13 @@ List<ReflectiveClassBuildItem> registerReflectiveClasses() {
         items.add(
                 ReflectiveClassBuildItem.builder("com.fasterxml.jackson.module.jakarta.xmlbind.JakartaXmlBindAnnotationModule")
                         .methods(false).fields(true).build());
-        items.add(ReflectiveClassBuildItem.builder(JsonNode.class).methods(false).fields(false).build());
+        items.add(ReflectiveClassBuildItem.builder("com.fasterxml.jackson.databind.JsonNode").methods(false).fields(false)
+                .build());
+        if (QuarkusClassLoader.isClassPresentAtRuntime("com.fasterxml.jackson.datatype.joda.JodaModule")) {
+            items.add(ReflectiveClassBuildItem.builder("com.fasterxml.jackson.datatype.joda.JodaModule").methods(false)
+                    .fields(false).build());
+            items.add(ReflectiveClassBuildItem.builder("org.joda.time.DateTime").methods(false).fields(false).build());
```

Comment:
`methods(false)` and `fields(false)` can be removed since they are `false` by default. I'm doing some clean works with 
- https://github.com/apache/camel-quarkus/issues/4694

Reply:
@claudio4j please rebase with `main` branch and I just merged https://github.com/apache/camel-quarkus/issues/4694. Sorry for the unconventince.